JAXRS/src/main/java/edu/psu/ist411/service/Model/Ingredient.java

```diff
@@ -0,0 +1,27 @@
+
+package edu.psu.ist411.service.Model;
```

Comment:
Package names should be lowercase. I don't think the instructor deducts points for this kind of thing so it's OK.